localstripe/resources.py

```diff
@@ -1527,19 +1529,35 @@ def __init__(self, url=None, limit=None):
         self.url = url
 
         self._limit = limit
+        self._starting_after = starting_after
+        self._starting_pos = 0
         self._list = []
 
     @property
     def data(self):
-        return [item._export() for item in self._list][:self._limit]
+        self._compute_starting_pos()
+        return [item._export() for item in self._list[
+            self._starting_pos:self._starting_pos + self._limit
+        ]]
 
     @property
     def total_count(self):
         return len(self._list)
 
     @property
     def has_more(self):
-        return len(self._list) > self._limit
+        self._compute_starting_pos()
+        return len(self._list) > self._limit + self._starting_pos
+
+    def _compute_starting_pos(self):
+        if self._starting_after is None:
+            return
+
+        for i, item in enumerate(self._list):
+            if getattr(item, 'id', None) == self._starting_after:
+                self._starting_pos = i + 1
+                self._starting_after = None
```

Comment:
@adrienverge this is the line I changed, so it only does the "heavy computation" once